common/src/main/java/com/wynntils/mc/render/FontRenderer.java

```diff
@@ -40,44 +40,42 @@ public int renderText(
             float x,
             float y,
             CustomColor customColor,
-            TextAlignment alignment,
+            HorizontalAlignment horizontalAlignment,
```

Comment:
There is already `FontRenderer#renderAlignedTextInBox` (although y alignment is only in #345)

Reply:
Im going to try and make a `FontRenderer#renderAlignedTextInBook` for both x and y, just x, and just y

Reply:
Code is in the settings pr already

Reply:
> There is already `FontRenderer#renderAlignedTextInBox` (although y alignment is only in #345)

There isn't a box here.

Reply:
[This](https://github.com/Wynntils/Artemis/pull/377/commits/747a9b0ccf0a498bbe4790f6103b4eef1064d73f) has added a few more `FontRenderer#renderAlignedTextInBox` methods, and should have the same signature (other than textalignment being deleted) as the new one in #345